source/PreferencesPanel.h

```diff
@@ -56,6 +56,12 @@ class PreferencesPanel : public Panel {
 
 	void Exit();
 
+	void HandleSettingsString(const std::string &str, Point cursorPosition);
```

Comment:
```suggestion
	void HandleSettingsString(const std::string &str, const Point &cursorPosition);
```
is this something we do? idk

Reply:
I mutate the value within the function, so I'd rather copy the value here.